dts/bindings/ethernet/qca,ar8031.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Qualcomm AR8031 PHY
+
+compatible: "qca,ar8031"
+
+include: phy.yaml
+
+properties:
+  reg:
+    required: true
+    description: PHY address
+
+  EEE:
```

Comment:
the name of this property is not descriptive, also should be lowercase

Reply:
fixed, thanks.